lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -82,9 +82,9 @@ describe('Fraggle Rock API', () => {
         notApplicableAudits,
       } = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`45`);
+      expect(auditResults.length).toMatchInlineSnapshot(`46`);
 
-      expect(notApplicableAudits.length).toMatchInlineSnapshot(`5`);
+      expect(notApplicableAudits.length).toMatchInlineSnapshot(`6`);
```

Comment:
Hmm INP shouldn't be N/A in this case. This inline snapshot is 5 on master.

Reply:
These tests really should be snapshoting the audit ids. As it is, it's too hard to debug. Does it matter?

Reply:
It might mean we aren't getting the events

Reply:
The test keeps timing out for me locally.